packages/schematics/src/collection/workspace/workspace.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { SchematicTestRunner } from '@angular-devkit/schematics/testing';
+import * as path from 'path';
+import { Tree, VirtualTree } from '@angular-devkit/schematics';
+import { getFileContent } from '@schematics/angular/utility/test';
+
+describe('workspace', () => {
+  const schematicRunner = new SchematicTestRunner(
+    '@nrwl/schematics',
+    path.join(__dirname, '../../collection.json')
+  );
+
+  let appTree: Tree;
+
+  beforeEach(() => {
+    appTree = new VirtualTree();
+  });
+
+  it('should error if no package.json is present', () => {
+    (expect as any).assertions(1);
```

Comment:
can we use expect().toThrow instead of `expect.assertions`? similar to how we do it in ngrx.spec.ts?

Reply:
Thanks, updated!